### DISCUSSION TITLE
Floating Tabs Bar

### DISCUSSION THREAD

**Discussion by w3ba1809** [quasarframework/quasar#11503] [General - Components / Directives / etc] [answered]
Hello guys, just asking if someone know how to achieve this funcionality in the quasar,

1. create a tabs bar like in the image
2.  basically you have a tabs bar buttons bellow of the app and when the user scrolls the bar tabs hide and when stops the bar tabs appears.

Thank your for any advice

![Screenshot_20211127_112739_com hugoapp client](https://user-images.githubusercontent.com/46602589/143691465-7441b10e-5aa2-4d21-a0dd-f8ca76496940.jpg)
e.

**Comment by pikil** [comment, marked as answer]
You can achieve it with q-footer and transparent background (hope the example I sketched would give an idea of what to do)

![Screen Shot 2021-12-01 at 10 10 49 AM](https://user-images.githubusercontent.com/26779040/144128652-e78eb2f1-31f3-4702-a812-e774d2005aab.png)


```
<template>
  <q-layout view="lHh Lpr lFf">
    <q-header elevated>
      <q-toolbar>
        <q-toolbar-title>TEST</q-toolbar-title>
        <div>Quasar v{{ $q.version }}</div>
      </q-toolbar>
    </q-header>
    <q-page-container>
      <router-view />
    </q-page-container>
    <q-footer class="bg-transparent q-pa-md">
      <q-tabs class="bg-white shadow-4" style="border-radius: 30px" indicator-color="primary">
        <q-tab class="q-pa-sm">
          <q-icon name="home" size="lg" color="primary" />
        </q-tab>
        <q-tab>
          <q-icon name="list" size="lg" color="primary" />
        </q-tab>
        <q-tab>
          <q-icon name="add" size="lg" color="primary" />
        </q-tab>
        <q-tab>
          <q-icon name="menu" size="lg" color="primary" />
        </q-tab>
      </q-tabs>
    </q-footer>
  </q-layout>
</template>
```

**Reply by w3ba1809** [reply]
Thank you so much for your time and example, that's what I need.
